docs/en/connector-v2/source/Http.md

```diff
@@ -181,6 +183,46 @@ connector will generate data as the following:
 |----------------------------------------------------------|
 | {"code":  200, "data":  "get success", "success":  true} |
 
+### keep_params_as_form
+For compatibility with old versions of http.
+When set to true,`<params>` and `<pageing>` will be submitted in the form.
+When set to false，`<params>` will be added to the url path,and `<pageing>` will be added to the body
```

Comment:
```suggestion
When set to false，`<params>` will be added to the url path,and `<pageing>` will not be added to the body or form. It will replace placeholders in params and body.
```